docs/sql-reference/sql-statements/data-manipulation/CREATE ROUTINE LOAD.md

```diff
@@ -44,6 +44,7 @@ Optional. The properties of the data. Syntax:
 [COLUMNS (<column1_name>[,<column2_name>,<column_assignment>,... ])],
 [WHERE <expr>],
 [PARTITION (<partition1_name>[,<partition2_name>,...])]
+[TEMPORARY PARTITION (<temporary_partition1_name>[,<temporary_partition2_name>,...])]
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/cdce615e-cbc9-458d-96c0-db9c06bbe218)
spaces are missing in between comma (,) and the left angle bracket (<)